README.md

```diff
@@ -15,6 +15,8 @@ Experimental REST API for APEL
 
 4. Navigate a web browser to "https://\<hostname\>/index/"
 
+The Docker Image does not enforce HTTPS connecitons due to it's primary deployment being on Indigo DataCloud infrastructure.
```

Comment:
fixed locally, will commit and push before merging (in case there other typos are spotted)